oam/install_scripts/post-install

```diff
@@ -234,31 +254,49 @@ if [ $user = "root" ]; then
 fi
 
 if [ $user = "root" ]; then
-	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	rm -f /etc/default/columnstore
+	$installdir/bin/syslogSetup.sh install > $tmpDir/syslog_install.log 2>&1
+
+	#check if MariaDB Columnstore system logging was setup
+	cat $tmpDir/syslog_install.log | grep 'No System Logging' >/dev/null 2>&1
+	if [ $? -eq 0 ]; then
+	    cat $tmpDir/syslog_install.log
+	fi
 else
-        sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstore.def
-        sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreLogRotate
+	sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreLogRotate
```

Comment:
I'm not so sure this works.  It might though.  Some regex processors are so comprensive it's possible to go a whole lifetime without seeing all of the syntax possibilities.  I've never seen '@' used like this.  I think the intended char is '/', which for sed is the delimiter between the args to the 's' command.